db/migrate/20160628222441_create_credentials.rb

```diff
@@ -4,7 +4,7 @@
   change do
     create_table :credentials do
       # Not an FK, because credentials won't be dropped when the RBAC is rebuilt
-      primary_key :role_id, type: String, null: false
```

Comment:
null:false in not needed in newer version?

Reply:
`primary_key: true` includes not null :-)